common/src/main/java/com/wynntils/wynn/model/scoreboard/objectives/ObjectiveListener.java

```diff
@@ -5,17 +5,17 @@
 package com.wynntils.wynn.model.scoreboard.objectives;
 
 import com.wynntils.core.WynntilsMod;
-import com.wynntils.wynn.model.scoreboard.ScoreboardHandler;
+import com.wynntils.handlers.scoreboard.ScoreboardListener;
+import com.wynntils.handlers.scoreboard.Segment;
 import com.wynntils.wynn.model.scoreboard.ScoreboardModel;
-import com.wynntils.wynn.model.scoreboard.Segment;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class ObjectiveHandler implements ScoreboardHandler {
+public class ObjectiveListener implements ScoreboardListener {
```

Comment:
Agree with Qodana, but I can't comment on line 76. 

I know I did this, but it is weird to have static fields, and a instance based listener.

Reply:
Doesn't have to be fixed here, though.